crypto/blake2/build.info

```diff
@@ -0,0 +1,3 @@
+LIBS=../../libcrypto
+SOURCE[../../libcrypto]=\
+        blake2b.c blake2s.c {- $target{blake2b_obj} -}
```

Comment:
That’s the point of code reviews :)  Yes, assembly is harder to integrate.  But do we think we're done now, from your viewpoint?